migrations/152_institutions__create.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE institutions (
+    id bigserial PRIMARY KEY,
+    institution TEXT NOT NULL,
+    uid_pattern TEXT
+);
+
+CREATE INDEX IF NOT EXISTS institutions_institution_idx ON institutions(institution);
+
+INSERT INTO institutions (id, institution, uid_pattern) VALUES
+    (1, 'UIUC', '%@illinois.edu'),
+    (2, 'ZJUI', '%@intl.zju.edu.cn'),
+    (3, 'gvsu.edu', '%@mail.gvsu.edu');
```

Comment:
I think we should just make this:
```
INSERT INTO institutions (id, institution) VALUES (1, 'Default');
```

Reply:
Then the prairielearn.engr.illinois.edu will need some entries when this goes live. But I agree with the change.